app/controllers/api/internal/two_factor_authentication/webauthn_controller.rb

```diff
@@ -35,6 +35,10 @@ def destroy
           analytics.webauthn_delete_submitted(**result.to_h)
 
           if result.success?
+            create_user_event(:webauthn_key_removed)
+            revoke_remember_device(current_user)
+            event = PushNotification::RecoveryInformationChangedEvent.new(user: current_user)
+            PushNotification::HttpPush.deliver(event)
```

Comment:
Separately, it might be worth thinking about extracting this behavior, since we're doing several things here, and we have the same code across basically all of MFAs... or at least I expect we intend to, but the RISC events aren't consistently implemented (even more justification to extract a common helper).

e.g.

```rb
module TwoFactorAuthenticatableMethods
  # ...
  def handle_valid_auth_method_removed(user_event:)
    create_user_event(user_event)
    revoke_remember_device(current_user)
    event = PushNotification::RecoveryInformationChangedEvent.new(user: current_user)
    PushNotification::HttpPush.deliver(event)
  end
end
```